src/containers/Transactions/test/reducer.test.js

```diff
@@ -1,6 +1,6 @@
 import * as actionTypes from '../actionTypes';
 import reducer, { initialState } from '../reducer';
-import mockTransaction from './mock_data/OfferCreate.json';
+import mockTransaction from '../../shared/components/Transaction/OfferCreate/test/mock_data/OfferCreateWithExpirationAndCancel.json';
```

Comment:
The scope of this is definitely too big for this PR, but I am strongly in favor of, if not moving the tests out to their own `tests` folder, then moving all the fixtures out to their own `fixtures` folder